config/routes.rb

```diff
@@ -4,10 +4,16 @@
     resources :categories
   end
 
+  root to: "home#index"
   resources :items, only: [:show, :index]
   resource :user, except: [:update]
   patch "/user", to: "users#update", as: :update_user
   get "login", to: "sessions#new"
   post "login", to: "sessions#create"
+  delete '/logout', to: 'sessions#destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.